torntools/scripts/content/displaycase/ttDisplayCase.js

```diff
@@ -0,0 +1,27 @@
+requireDatabase().then(() => {
+	console.log("TT - Display Case");
+	displayCabinetWorth();
+});
+
+function displayCabinetWorth() {
+	if (settings.pages.displaycase.worth) {
+		let display_user_id = window.location.href.split("/")[4];
+		fetchApi_v2("torn", {
+			section: "user",
+			objectid: display_user_id,
+			selections: "display",
+		}).then((result) => {
+			let totalValue = 0;
+			for (let item in result.display) {
+				totalValue +=
+					result.display[item].market_price *
+					result.display[item].quantity;
+			}
+			doc.find(
```

Comment:
I would prefer to create a new element with this HTML and then appending it. Just because Firefox throws warnings when we use `.innerHTML`. It doesn't break anything, just something to try to follow.

Reply:
Changed.